apps/src/templates/MultiItemInput.tsx

```diff
@@ -31,28 +41,31 @@ export const MultiItemInput: React.FunctionComponent<{
                 multiline ? moduleStyles.textarea : moduleStyles.inlineLabel
               )}
               onChange={e => onChange(index, e.target.value)}
+              disabled={readOnly}
             />
           );
         })}
       </div>
-      <div className={moduleStyles.buttonsRow}>
-        {showPlus && (
+      {!readOnly && (
+        <div className={moduleStyles.buttonsRow}>
+          {showPlus && (
+            <button
```

Comment:
What do you think about using design system buttons for these now? When I was doing the levelbuilder edit page I kept the UI pretty minimal but now that this is student facing, might be worth tweaking the presentation on these a bit?

Reply:
I actually don't see any mocks for this at all :) Updated to use the new `Button` component, and used the "secondary" styling that we use for the "Add" button when adding retrieval items. I've generally been trying to follow the guidance Jessie shared on Monday at sprint planning to focus efforts on getting things working end to end (ie, avoiding sinking too much time into styling perfection just yet)

![image](https://github.com/code-dot-org/code-dot-org/assets/25372625/49409a32-04cb-4524-9f5d-cc28b60cfbec)